src/Umbraco.Web.UI.Client/src/common/directives/validation/valserver.directive.js

```diff
@@ -7,7 +7,7 @@
     **/
 function valServer(serverValidationManager) {
     return {
-        require: ['ngModel', '?^^umbProperty'],
```

Comment:
@nielslyngsoe There was a reason that this was optional, not sure why you changed it? If it's not optional we end up with js errors when loading the content type editor because property editors are loaded in their to be previewed.